### PR TITLE
chore: update auto labels

### PR DESCRIPTION
**Pull request by weihanglo**
See

* `A-errors` has been renamed to `A-diagnostics`
* Some manifest related modules were moved to `cargo-util-schemas`.

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -120,8 +120,7 @@ trigger_files = [
   "src/cargo/util/config/environment.rs",
 ]
 
-[autolabel."A-errors"]
-trigger_files = ["src/cargo/util/diagnostic_server.rs"]
```
Comment (ehuss) [inline triagebot.toml:124]: This looks like it deleted the trigger, was that intentional?

Comment (weihanglo) [reply triagebot.toml:124]: I feel like that `diagnostic_server.rs` should belong to `Command-fix`, no?

Comment (ehuss) [reply triagebot.toml:124]: Sorry, I was not clear. I was confused that this is adding this empty table:

```
[autolabel."A-diagnostics"]
```

for `A-diagnostics` without any keys. If that table is not needed, I'm not clear why it is being added (I wasn't sure if there was a `trigger_files` intended to be added here).

Feel free to `r=ehuss` either way.

**Comment by weihanglo** [comment]
`A-errors` auto-label has been removed.

@bors r=ehuss

**Pull request merged**